tests/layer_tests/tensorflow2_keras_tests/test_tf2_keras_activation.py

```diff
@@ -50,8 +50,8 @@ def create_keras_activation_net(self, activation_func, input_names, input_shapes
              input_type=tf.float32),
         dict(activation_func="swish", input_names=["x1"], input_shapes=[[5, 4, 8]],
              input_type=tf.float32),
-        dict(activation_func="tanh", input_names=["x1"], input_shapes=[[5, 4, 8, 3]],
-             input_type=tf.float32),
+        pytest.param(dict(activation_func="tanh", input_names=["x1"], input_shapes=[[5, 4, 8, 3]],
+                          input_type=tf.float32), marks=pytest.mark.skip(reason="110006")),
```

Comment:
what's wrong with this test as well as conv lstm one?

Reply:
Looks internal problem in TF itself. I will investigate this in the ticket.

Reply:
So, this isn't connected to the changes you are making, right? Why bother including them in this PR then. A bit confusing.

Reply:
yes, right. We just changed interface of TF inference with named inputs and it looks broken there.